packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -86,7 +67,15 @@ const liquidate = async (
   ]);
   // NB: all the proceeds from AMM sale are on the vault seat instead of a staging seat
 
-  const { shortfall, overage } = discrepancy(debt, proceeds.RUN);
+  const [overage, shortfall] = AmountMath.isGTE(debt, proceeds.RUN)
```

Comment:
I'm not sure this is better. read better before, imho.

Reply:
either way is OK to me.

Reply:
I might have gone with

```
  const short = AmountMath.isGTE(debt, proceeds.RUN);
  const overage = short
    ? AmountMath.makeEmptyFromAmount(debt)
    : AmountMath.subtract(proceeds.RUN, debt);
  const shortfall = short
    ? AmountMath.subtract(debt, proceeds.RUN)
    : AmountMath.makeEmptyFromAmount(debt);
```
I'm not a fan of the trinary assigning two values